expedited-procedure.md

```diff
@@ -41,6 +41,14 @@ Cross functional with the following elements:
 
 For out of hours use the [web contact details](https://holidayextras.jira.com/wiki/display/WEB/Web+Contact+Details) for contact info.
 
+## Communication
+
+Regular communication is essential. By sending regular updates the Web Team and HX stakeholders are able to manage and mitigate the impact of the issue.
+
+* Post in the main [#expedite](https://holidayextras.slack.com/messages/expedite/) slack channel.
+* If issue is complex enough consider a dedicated Slack channel. Although regular communication in the primary #expedite channel should still continue.
+* If the issue is going to effect deployments, post in the [#deployments](https://holidayextras.slack.com/messages/deployments/) Slack channel.
```

Comment:
Also notify the pest controller for the week in the `#pest-control` channel to avoid any duplicated effort? Or should the pest controller be following #expedite anyway?